pdi-dataservice-client-api/pom.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>pentaho</groupId>
+    <artifactId>pdi-dataservice-client</artifactId>
+    <version>8.1.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>pdi-dataservice-client-plugin-api</artifactId>
+  <version>8.1.0.0-SNAPSHOT</version>
```

Comment:
This version is not needed, you can inherit it from the parent pom.

Reply:
Right! Thanks